cli/mender-artifact/main.go

```diff
@@ -543,6 +543,12 @@ func getCliContext() *cli.App {
 			Name:  "print-cmdline",
 			Usage: "Print the command line that can recreate the same Artifact with the components being dumped. If all the components are being dumped, a nearly identical Artifact can be created. Note that timestamps will cause the checksum of the Artifact to be different, and signatures can not be recreated this way. The command line will only use long option names.",
 		},
+		cli.StringFlag{
+			Name: "separator",
+			Usage: "Choose the separator to replace whitespace with for the 'print-cmdline' flag (default: <space>)",
```

Comment:
Unfortunately being able to choose the separator won't work they way I had intended, because you cannot specify a nul character. What I had in mind was how the `-print0` operator works in [`find`](https://linux.die.net/man/1/find). The reason nul is particularly handy is that `xargs` supports the `-0` character which reads arguments terminated by nul characters, which would be very useful for shell handling. What about simply `print0-cmdline`, to mirror that?

Reply:
Alright, I misunderstood this a little